doc/api/errors.md

```diff
@@ -795,6 +796,76 @@ Used when an attempt is made to launch a Node.js process with an unknown
 by errors in user code, although it is not impossible. Occurrences of this error
 are most likely an indication of a bug within Node.js itself.
 
+<a id="ERR_MULTIPLE_CALLBACK"></a>
+### ERR_MULTIPLE_CALLBACK
+
+Used when a callback is called more than once.
+
+<a id="ERR_ASSERTION"></a>
+### ERR_ASSERTION
+
+Used as special type of error that can be triggered whenever Node.js detects an
+exceptional logic violation that should never occur. These are raised typically
+by the `assert` module.
+
+<a id="ERR_CPU_USAGE"></a>
+### ERR_CPU_USAGE
+
+Used when the native call from process.cpuUsage can't be processed properly.
+
+<a id="ERR_HTTP_HEADERS_SENT"></a>
+### ERR_HTTP_HEADERS_SENT
+
+Used in case the headers were already sent and another attempt is made to add
+more headers.
+
+<a id="ERR_INVALID_CHAR"></a>
+### ERR_INVALID_CHAR
+
+Used in e.g. http if invalid characters are detected in headers.
+
+<a id="ERR_HTTP_INVALID_STATUS_CODE"></a>
+### ERR_HTTP_INVALID_STATUS_CODE
+
+Used for nonexisting status codes.
```

Comment:
I am aware that the validation is less strict but personally I would keep the description like that as it feels more intuitive to me. With the range I would wonder why that range was chosen. As far as I know the current bounds were chosen to be "somewhat" near the regular status codes.